src/Extension/CoreExtension.php

```diff
@@ -1693,8 +1693,12 @@ function twig_array_batch($items, $size, $fill = null, $preserveKeys = true)
     return $result;
 }
 
-function twig_array_filter($array, $arrow)
+function twig_array_filter(Environment $env, $array, $arrow)
 {
+    if ($env->getExtension('\Twig\Extension\SandboxExtension')->isSandboxed() && !$arrow instanceof Closure) {
```

Comment:
the leading `\` should be removed, to go through the fast path where the exact class name is provided (`get_class()` in PHP does not return a leading namespace separator)

Reply:
I added it for consistency with all other calls to `getExtension` and `hasExtension` in the project.

The performance impact is negligible, as the first instruction of getExtension is `$class = ltrim($class, '\\');`

Reply:
hmm, I forgot we un-conditionnally trim the char, without first checking for the key as is.